packages/jsii-pacmak/tsconfig.json

```diff
@@ -48,7 +48,19 @@
     // "inlineSources": false,                    /* Emit the source alongside the sourcemaps within a single file; requires '--inlineSourceMap' or '--sourceMap' to be set. */
 
     /* Experimental Options */
-    "experimentalDecorators": true            /* Enables experimental support for ES7 decorators. */
+    "experimentalDecorators": true,           /* Enables experimental support for ES7 decorators. */
     // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
-  }
+    "composite": true                         /* Allow this project to be referenced by others. */
+  },
+  "include": [
+    "**/*.ts"
+  ],
+  "exclude": [
+    "**/*.d.ts",
+    "node_modules"
+  ],
+  "references": [
```

Comment:
What happens when someone needs to add a dependency? Do they have to remember to add it here?

Reply:
It should work without being added in `references`. If it's a dependency from the same monorepo (aka you have the `.ts` files available, not just the `.d.ts`), then you'll get sub-optimal compile times.

Reply:
got it... did you see any improvement?

Reply:
It feels faster, but it's a small project so it's hard to tell for sure...

Reply:
TBH feels like a redundant smartassness to me... More configuration, more ways for things to go wrong, harder to understand what's going on... If you are keen on this, be my guest, but in my view if there is no clear benefit, KISS!

Reply:
Yeah, so I'll be generating project references for JSII assemblies' managed `tsconfig.json`, but I'm removing the support for the `jsii` monorepo packages (it doesn't work when using references *in-tree*, and the error message... isn't helpful).

Reply:
Not sure I understand and it's worrying me. Let's talk about it tomorrow?

Reply:
We can talk about this over the PR that I'll submit a bit later that'll create those automatically. No need to make a fuss now :)

Reply:
I've already started crying 😢